python/python.md

```diff
@@ -253,6 +260,8 @@ print 'always executed'
 ### Independent Practice
 
 - Create a function that will take a string argument and return that string in all uppercase letters.
+	def uppercase(str):
```

Comment:
This is something the learner is supposed to solve - don’t give them the answer!